### PR TITLE
feat(relayer): Add support for ACX

### PR DESCRIPTION
**Pull request by kevinuma**

### CODE REVIEW

File: src/clients/bridges/ArbitrumAdapter.ts [hunk 1]
```diff
@@ -25,6 +25,7 @@ const l1Gateways = {
   "0x04Fa0d235C4abf4BcF4787aF4CF447DE572eF828": "0xa3A7B6F88361F48403514059F1F16C8E78d60EeC", // UMA
   "0x3472A5A71965499acd81997a54BBA8D852C6E53d": "0xa3A7B6F88361F48403514059F1F16C8E78d60EeC", // BADGER
   "0xba100000625a3754423978a60c9317c58a424e3D": "0xa3A7B6F88361F48403514059F1F16C8E78d60EeC", // BAL
+  "0x53691596d1BCe8CEa565b84d4915e69e03d9C99d": "0xa3A7B6F88361F48403514059F1F16C8E78d60EeC", // ACX
```
Comment (nicholaspai) [inline src/clients/bridges/ArbitrumAdapter.ts:28]: This seems like the wrong L1 address

File: src/clients/bridges/ArbitrumAdapter.ts [hunk 2]
```diff
@@ -38,6 +39,7 @@ const l2Gateways = {
   "0x04Fa0d235C4abf4BcF4787aF4CF447DE572eF828": "0x09e9222E96E7B4AE2a407B98d48e330053351EEe", // UMA
   "0x3472A5A71965499acd81997a54BBA8D852C6E53d": "0x09e9222E96E7B4AE2a407B98d48e330053351EEe", // BADGER
   "0xba100000625a3754423978a60c9317c58a424e3D": "0x09e9222E96E7B4AE2a407B98d48e330053351EEe", // BAL
+  "0x53691596d1BCe8CEa565b84d4915e69e03d9C99d": "0x09e9222E96E7B4AE2a407B98d48e330053351EEe", // ACX
```
Comment (nicholaspai) [inline src/clients/bridges/ArbitrumAdapter.ts:42]: Same issue, wrong L1 address

File: src/clients/bridges/ArbitrumAdapter.ts (hunk 1 above)
Comment (kevinuma) [reply src/clients/bridges/ArbitrumAdapter.ts]: Nice catch! I incorrectly used the Arb address. Updated now.

File: src/clients/bridges/ArbitrumAdapter.ts (hunk 2 above)
Comment (kevinuma) [reply src/clients/bridges/ArbitrumAdapter.ts]: Updated

**Review by nicholaspai — APPROVED** [review]
LGTM, the config store is updated now so we can start supporting relays

**Review by mrice32 — APPROVED** [review]
LGTM!

**Pull request merged**